src/main/java/frc/robot/subsystems/Dashboard.java

```diff
@@ -0,0 +1,335 @@
+        _lowerVelocity = shooterFlywheel.add("Lower Velocity", 0).withPosition(0, 1).withSize(1, 1).withWidget(BuiltInWidgets.kNumberBar).withProperties(Map.of("Min", -6000, "Max", 6000)).getEntry();
+        // Autonomous Options
+        var autonomousLayout = tab.getLayout("Autonomous", BuiltInLayouts.kGrid).withPosition(30, 0).withSize(9, 8).withProperties(Map.of("Number of columns", 1, "Number of rows", 4, "Label position", "LEFT"));
+
+        // Autonomous delay chooser setup
+        _autoDelayChooser = new SendableChooser<>();
+        _autoDelayChooser.setDefaultOption("0", 0);
+        _autoDelayChooser.addOption("1", 1);
+        _autoDelayChooser.addOption("2", 2);
+        _autoDelayChooser.addOption("3", 3);
+        _autoDelayChooser.addOption("4", 4);
+        _autoDelayChooser.addOption("5", 5);
+        autonomousLayout.add("Delay Options", _autoDelayChooser).withPosition(0, 0).withSize(1, 1).withWidget(BuiltInWidgets.kSplitButtonChooser);
+
+        // Autonomous start position chooser setup
+        // _autoStartPositionChooser = new SendableChooser<>();
+        // _autoStartPositionChooser.setDefaultOption("Middle",
+        // DrivePosition.MiddleStart);
+        // _autoStartPositionChooser.addOption("Substation Side",
+        // DrivePosition.SubstationStart);
+        // _autoStartPositionChooser.addOption("Wall Side", DrivePosition.WallStart);
+        // autonomousLayout.add("Start Position", _autoStartPositionChooser)
+        // .withPosition(0, 1)
+        // .withSize(1, 1)
+        // .withWidget(BuiltInWidgets.kComboBoxChooser);
+
+        var outerLayout = SettingsTab.getLayout("Settings", BuiltInLayouts.kGrid).withPosition(0, 0).withSize(40, 10).withProperties(Map.of("Number of columns", 6, "Number of rows", 1, "Label position", "TOP"));
+
+        var driveSettings = outerLayout.getLayout("Drive Subsystem", BuiltInLayouts.kList).withPosition(0, 0).withSize(8, 10).withProperties(Map.of("label position", "LEFT"));
+        _flOffset = driveSettings.add("FL Offset", 0.0).withPosition(0, 0).withWidget(BuiltInWidgets.kTextView).getEntry();
+        _frOffset = driveSettings.add("FR Offset", 0.0).withPosition(2, 0).withWidget(BuiltInWidgets.kTextView).getEntry();
+        _blOffset = driveSettings.add("BL Offset", 0.0).withPosition(0, 2).withWidget(BuiltInWidgets.kTextView).getEntry();
+        _brOffset = driveSettings.add("BR Offset", 0.0).withPosition(2, 2).withWidget(BuiltInWidgets.kTextView).getEntry();
+
+        var IntakeSetting = outerLayout.getLayout("Intake", BuiltInLayouts.kList).withPosition(1, 0).withSize(7, 10).withProperties(Map.of("label position", "LEFT"));
+        _IntakeInSpeed  = IntakeSetting.add("Intake In Speed", 0.0).withPosition(0, 0).withWidget(BuiltInWidgets.kTextView).withPosition(0, 0).getEntry();
+        _IntakeOutSpeed = IntakeSetting.add("Intake Out Speed", 0.0).withWidget(BuiltInWidgets.kTextView).withPosition(0, 1).getEntry();
```

Comment:
1. Updated tab name
2. Updated layout name
3. Updated variable names
4. Updated defaults to use constants
5. Removed `withSize` (in a grid layout)
6. Removed `withPosition` for the entries (in a list layout)

```suggestion
        var intakeSettings = outerLayout.getLayout("Intake", BuiltInLayouts.kList).withPosition(1, 0).withProperties(Map.of("label position", "LEFT"));
        _intakeInSpeed  = intakeSettings.add("Intake In Speed", Constants.Intake.INTAKE_DEFAULT_PERCENT_OUTPUT).withWidget(BuiltInWidgets.kTextView).getEntry();
        _intakeOutSpeed = intakeSettings.add("Intake Out Speed", Constants.Intake.OUTTAKE_DEFAULT_PERCENT_OUTPUT).withWidget(BuiltInWidgets.kTextView).getEntry();
```